.github/workflows/deploy.yml

```diff
@@ -1,9 +1,8 @@
 name: Deploy website to IBM Cloud
 
 on:
-  push:
-    branches:
-      - master
+  schedule:
+    - cron: '0 8 * * *'
```

Comment:
2am central time

Reply:
[Fun site](https://crontab.guru/#0_8_*_*_*) 👀

Reply:
I used that!! Haha